test-monitor.cpp

```diff
@@ -1,8 +1,82 @@
-#include "./monitor.h"
-
+﻿#include "./monitor.h"
 #include "gtest/gtest.h"
+#include "Localization.h"
+
+// Initialize translations for different languages
+void initializeLocalization() {
+    Localization& localization = Localization::getInstance();
+    localization.loadTranslations("en", {

```

Comment:
Nice abstraction to represent localized messages 👍 
However, the strings aren't automatically tested. Also, it's hard to test all the situations in all the languages manually each time.
Of course, the "understand-ability" of the translation can only be confirmed manually by native speakers.
But other things can go wrong, like missing a translation in a particular language, etc.
Can you think of any asserts to safeguard against that?